crypto/openstark/src/proof_params.rs

```diff
@@ -45,3 +45,91 @@ pub struct ProofParams {
     /// to provide the right number points.
     pub constraints_degree_bound: usize,
 }
+
+// A default un-optimized 50 bit secure system.
+#[allow(dead_code)]
+fn params_suggestion(constraints_degree_bound: usize) -> ProofParams {
+    ProofParams {
+        blowup: 16,
+        pow_bits: 12,
+        queries: 20,
+        fri_layout: vec![3, 2],
```

Comment:
Doesn't fri-layout depend on trace-table size?

Reply:
In that 2 + sum(fri_layout) can't be greater than log_2(trace_table_size), but I'm not sure how else.